arcgis-ios-sdk-samples/Utility Network/Perform valve isolation trace/PerformValveIsolationTrace.storyboard

```diff
@@ -0,0 +1,116 @@
+                                </constraints>
+                                <viewLayoutGuide key="safeArea" id="C4m-2x-Lci"/>
+                            </view>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="mcq-w1-aYB">
+                                <rect key="frame" x="0.0" y="818" width="414" height="44"/>
+                                <items>
+                                    <barButtonItem style="plain" id="w6l-Ji-WPD">
+                                        <view key="customView" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="5Ke-ZQ-uR3">
+                                            <rect key="frame" x="20" y="22" width="130" height="0.0"/>
+                                            <subviews>
+                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Isolated  Features" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="3" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="lHG-2p-x4N">
+                                                    <rect key="frame" x="0.0" y="-17" width="70" height="34"/>
+                                                    <constraints>
+                                                        <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="70" id="Yay-jv-ubm"/>
+                                                    </constraints>
+                                                    <fontDescription key="fontDescription" type="system" pointSize="14"/>
+                                                    <nil key="textColor"/>
+                                                    <nil key="highlightedColor"/>
+                                                </label>
+                                                <switch opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" contentHorizontalAlignment="center" contentVerticalAlignment="center" on="YES" translatesAutoresizingMaskIntoConstraints="NO" id="o40-Kg-nZ5">
+                                                    <rect key="frame" x="75" y="0.0" width="51" height="0.0"/>
+                                                </switch>
+                                            </subviews>
+                                            <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                            <constraints>
+                                                <constraint firstItem="o40-Kg-nZ5" firstAttribute="top" relation="greaterThanOrEqual" secondItem="5Ke-ZQ-uR3" secondAttribute="top" id="AwD-XY-htV"/>
```

Comment:
Top and bottom constraints shouldn't be necessary here as the height of the view should be determined by the toolbar.

Reply:
There was an issue similar to this: https://stackoverflow.com/questions/45854203/uibarbuttonitem-not-clickable-on-ios-11-beta-7

It is related to putting custom `UIView` in `UIToolbar`. Autolayout will give the custom view a 0x0 frame, which covers the tap area of the `UISwitch`. To avoid this, now I simply changed the design and put the switch in another `UIBarButtonItem`.

Reply:
Oh weird! I saw that it displayed properly, but I didn't try toggling the switch.